Asterix/tests/test_coronagraph.py

```diff
@@ -14,11 +14,12 @@ def test_all_coronagraphs():
     # Reassign the parameter groups to variables
     modelconfig = config["modelconfig"]
     Coronaconfig = config["Coronaconfig"]
+    Coronaconfig.update({"filename_instr_apod": "RoundPup"})
 
     # Set coronagraph to be tested
     coros_to_test = ["fqpm", "wrapped_vortex", "classiclyot", "knife", "hlc", "vortex"]
-    expected_attenuation = [1e-20, 8e-10, 5e-3, 1e-2, 1e-3, 1e-20]  # Note that these are for the 100 px pupil below
-    atols = [0, 1e-14, 1e-14, np.nan, 1e-14, 0]  # zeros are for perfect coronagraphs
+    expected_attenuation = [1e-20, 5e-9, 5e-3, 1e-2, 1e-3, 8e-9]  # Note that these are for the 256 px pupil below
```

Comment:
Why did it get worse for the WV?

Reply:
Because I've greatly reduced the pixel size of the focal plane and pupil plane to go faster

- in current master : Size of the pupil is 256 before padding and 512 after padding, size of the FP (same as dimScience) is 400: 
Attenuation of 'wrapped_vortex is 5.357987020188405e-10 time for the corono propagation is 0.01989s
- with this PR Size of the pupil is 200 and 300 after padding, size of the FP is 256:
Attenuation of 'wrapped_vortex is 2.081871059763059e-09  time for the corono propagation is 0.00591s

Time is not done super properly because only for 1 corono propagation but gives an order or idea (faster by a facteur 2-3)

I decided that 2 10-9 is good enough for our purpose because this is the kind of contrast limit we want to get on the testbed. As you know, it's annoying to optimize because it is not super reliable, increasing the number of pixels sometimes get you worse contrast, or maybe you need to better optimize the number of element of resolution in the list each time you increase the number of pixel. I might MCMC the stuff and try to get the best contrast for both pupil plane and focal plane size under 256. Feel free to play with it if you have an insight on it.

Reply:
Understood, thanks!